docs/.eslintrc.js

```diff
@@ -1,29 +1,104 @@
 module.exports = {
   root: true,
-  parser: 'vue-eslint-parser',
-  plugins: ['gridsome'],
-  env: {
-    node: true,
-  },
+  plugins: ['prettier', 'node', 'jest', 'jsdoc'],
```

Comment:
Why the `gridsome` eslint [plugin](https://eslint.gridsome.org/introduction/) is removed here? 
seems to be very useful to check gridsome specifics

Reply:
yarn 2 issues revolving round the plugin. So there is a great need to remove `eslint-plugin-gridsome` and these plugins are self sufficient to manage all the specifics.